integration-cli/docker_cli_pull_test.go

```diff
@@ -108,10 +106,8 @@ func (s *DockerHubPullSuite) TestPullAllTagsFromCentralRegistry(c *check.C) {
 
 	s.Cmd(c, "pull", "--all-tags=true", "busybox")
 	outImageAllTagCmd := s.Cmd(c, "images", "busybox")
-	if linesCount := strings.Count(outImageAllTagCmd, "\n"); linesCount <= 2 {
-		c.Fatalf("pulling all tags should provide more images, got %d", linesCount-1)
-	}
-
+	linesCount := strings.Count(outImageAllTagCmd, "\n")
+	c.Assert(linesCount <= 2, checker.Equals, false, check.Commentf("pulling all tags should provide more images, got %d", linesCount-1))
```

Comment:
`GreaterThan` / `LessThan` are now in master, if you rebase, you can use them :wink